### DISCUSSION TITLE
Is there a way to make changing focused "absolute"

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#6231] [Q&A]
By default, when you press `mod+left`, it moves to the last active window on the left container. Is there a way to make it just focus the window that is visually the the left of the current window?

Right now, having to 4 window open in a grid and moving left and right often causes focus to shift from top right to bottom left when pressing `mod+left` for example, which I find a bit confusing to use. Can you force it to move from top right to top left, even if the bottom left window was the last focused window on the left?

**Comment by yusufaktepe** [comment]
```
focus_wrapping no
```

See: https://i3wm.org/docs/userguide.html#_focus_wrapping

**Reply by ghost** [reply]
> ```
> focus_wrapping no
> ```
> 
> See: https://i3wm.org/docs/userguide.html#_focus_wrapping

That doesn't seem to change it, pressed right will still go to the last used window on the right, not the one visually to the right of the current window

**Reply by yusufaktepe** [reply]
Ah, right. I misunderstood the question.

I don't think there is a config for that.

**Reply by ghost** [reply]
> Ah, right. I misunderstood the question.
> 
> I don't think there is a config for that.

No problem, I found a Python script which does what I was thinking: [i3-tools](https://github.com/miseran/i3-tools) (window-tool in particular).
